doc/source/multivisit_api.rst

```diff
@@ -0,0 +1,59 @@
+.. _multivisit_api:
+****************************
+API for MVM Processing Code
+****************************
+
+The ``hapmultisequencer.py`` module is part of the drizzlepac package and calls
+utilities which are intended to be used with
+Multi-visit Mosaic (MVM) data and to be imported into other Python programs or
+a Python session for interactive data analysis.
+
+.. toctree::
+   :maxdepth: 1
+
+hapmultisequencer
+---------------------
+Primary user interface for performing MVM processing.
+
```

Comment:
This line of text is just hanging out in space.  It could be part of the section title or made into a sentence and combined into the first paragraph.

Reply:
Fixed in 3cd7f7e.